blog/2023-08-22-nodejs_postgres/index.md

```diff
@@ -0,0 +1,237 @@
+---
+slug: how-to-get-started-with-nodejs-postgres
+title: How to get started with NodeJs + Postgres Datasource on the Humalect platform
```

Comment:
How to Deploy NodeJS Application with Postgres Datasource on Azure (in less than 10 mins)?